sentry_sdk/integrations/django/signals_handlers.py

```diff
@@ -7,12 +7,11 @@
 from sentry_sdk._functools import wraps
 from sentry_sdk._types import TYPE_CHECKING
 from sentry_sdk.consts import OP
+from sentry_sdk.integrations.django import DJANGO_VERSION
 
 
 if TYPE_CHECKING:
-    from typing import Any
-    from typing import Callable
-    from typing import List
+    from typing import Any, Callable, List, Tuple, Union
```

Comment:
`typing.List` and `typing.Tuple` are also deprecated since 3.9 in favor of the builtin `list` and `tuple`, respectively.

Reply:
Updated.